pages/common/steamcmd.md

```diff
@@ -0,0 +1,16 @@
+# steamcmd
+
+> A command-line version of the Steam client.
+> More information: <https://manned.org/steamcmd>.
+
+- Install or update an application anonymously:
+
+`steamcmd +login {{anonymous}} +app_update {{appid}} +quit`
+
+- Install or update an application using the specified credentials:
+
+`steamcmd +login {{username}} {{password}} +app_update {{appid}} +quit`
```

Comment:
Is it also possible to leave out the password and being promted interactively? We try to never encourage anyone to put a password in a command, which ends up in the history.

Reply:
Checked it now, turns out if one omits the password field that it will prompt for it interactively.

> `login <username>`
> \- https://developer.valvesoftware.com/wiki/SteamCMD#With_a_Steam_account